src/modules/ProductLicense.rb

```diff
@@ -1182,11 +1182,10 @@ def SearchForLicense_NormalRunBaseProduct(_src_id, fallback_dir)
     def SearchForLicense_AddOnProduct(src_id, _fallback_dir)
       Builtins.y2milestone("Getting license info from repository %1", src_id)
 
-      @info_file = Pkg.SourceProvideDigestedFile(
+      @info_file = Pkg.SourceProvideOptionalFile(
         src_id, # optional
         1,
-        "/media.1" + INFO_FILE,
-        true
+        INFO_FILE
```

Comment:
Actually I find it a bit confusing, in the registration we remove the info file handling completely and here the code is left, just changing the `INFO_FILE` value to `README.BETA`. 

That's confusing, the methods/variables mean something completely different than the name suggests. I'd rename the variables/methods (except the `InstShowInfo` module methods not not break the API)...

Reply:
It makes sense.